src/components/views/settings/tabs/user/HelpUserSettingsTab.js

```diff
@@ -80,6 +80,14 @@ export default class HelpUserSettingsTab extends React.Component {
         });
     };
 
+    _onMarkAllRoomsAsRead = (e) => {
+        MatrixClientPeg.get().getRooms().forEach(function(element) {
+            MatrixClientPeg.get().sendReadReceipt(function(evs) {
+                return evs[evs.length - 1]
```

Comment:
The linter is annoying sometimes:
```suggestion
                return evs[evs.length - 1];
```